packages/position/src/react/index.tsx

```diff
@@ -160,7 +160,10 @@ export const Position = React.forwardRef<HTMLElement, PositionProps>(
             )}
 
         {createUniversalPortal(
-          <div style={{ visibility: shownOnce ? 'visible' : 'hidden' }}>
+          <div
+            style={{ display: shownOnce ? 'initial' : 'none' }}
+            aria-live="polite"
```

Comment:
is this to limit the amount of updates read out to the user while scrolling?

Reply:
I wanted to let the screen reader know something will eventually appear here after a user interaction. Initially on mount this is an empty div. We should let assistive technology know its dynamic and may render content at some point (politely).